maid.gemspec

```diff
@@ -36,7 +36,7 @@ Gem::Specification.new do |s|
   s.add_dependency('mime-types', '~> 3.0', '< 4.0')
   s.add_dependency('rubyzip', '~> 2.3.2')
   s.add_dependency('rufus-scheduler', '~> 3.8.2')
-  s.add_dependency('thor', '~> 1.2.1')
+  s.add_dependency('thor', '< 2')
```

Comment:
Could you please change to `~> 1.4.0` to keep in line with Ruby conventions and explicitly state the minimum version?